src/appleseed-max-impl/appleseedrenderer/renderersettings.cpp

```diff
@@ -58,9 +58,25 @@ namespace
 
             m_gi = true;
             m_caustics = false;
-            m_bounces = 3;
+            m_bounces = 8;
+            m_dbounces = 3;
```

Comment:
I would probably use more complete name, like `m_max_diffuse_bounces`. Or at least `m_diffuse_bounces".

Reply:
`m_bounces` was there already, I have not introduced it. It refers to the global bounce limit. I changed the default to 8, initially it was set to 3 which is too low. 
But I agree that with more settings getting added we should make some of them more verbose.